src/ui/public/timefilter/timefilter.js

```diff
@@ -108,7 +109,35 @@ uiModules
     };
 
     Timefilter.prototype.getActiveBounds = function () {
-      if (this.enabled) return this.getBounds();
+      if (this.isTimeRangeSelectorEnabled) return this.getBounds();
```

Comment:
How about updating the style to include `{ }` while touching this? (consider the `git blame` karma :wink:)